src/Vega4Wrapper.js

```diff
@@ -0,0 +1,207 @@
+var makeValidator = require('domain-validator');
+
+class Vega4Wrapper {
+    /**
+     * Shared library to wrap around vega code
+     * @param {Object} wrapperOpts Configuration options
+     * @param {Object} wrapperOpts.loader Vega-loader object, its sanitize(), file() and load() method will be overwrited
+     * @param {Object} wrapperOpts.domains allowed protocols and a list of their domains
+     * @param {Object} wrapperOpts.domainMap domain remapping
+     * @param {Function} wrapperOpts.logger
+     * @param {Function} wrapperOpts.formatUrl
+     * @param {string} [wrapperOpts.languageCode]
+     * @constructor
+     */
+    constructor(wrapperOpts) {
+        var self = this;
+        // Copy all options into the wrapper
+        Object.assign(self, wrapperOpts);
+        self.validators = {};
+
+        self.loader.sanitize = self.sanitize.bind(self);
+        self.loader.load = (uri, options) => {
```

Comment:
no need for return -- remove the `{` and `return`

Reply:
Are you sure? Then the interface will differ from the origin and document.
![image](https://user-images.githubusercontent.com/49155264/57963277-81003900-7954-11e9-91c5-837d6953a03f.png)

Reply:
how would it differ??  Instead of your code, it will be this:
```
this.loader.load = (uri, options) => 
   this.sanitize(uri, options)
       .then(opt => {
                    var url = opt.href;
```

Reply:
The load function has a return value, which is a Promise.